test/fixtures/SpokePool.Fixture.ts

```diff
@@ -175,19 +175,19 @@ export function getRelayHash(
   _originChainId: number,
   _destinationChainId: number,
   _destinationToken: string,
-  _amount?: string,
-  _realizedLpFeePct?: string,
-  _relayerFeePct?: string
+  _amount?: BigNumber,
+  _realizedLpFeePct?: BigNumber,
+  _relayerFeePct?: BigNumber
```

Comment:
this is the reason all the other `.toStrings()` were removed: we now use the correct types.